runbotics-orchestrator/src/main/java/com/runbotics/service/UserService.java

```diff
@@ -374,6 +374,12 @@ public Page<AdminUserDTO> getAllManagedUsers(Pageable pageable) {
         return userRepository.findAll(pageable).map(AdminUserDTO::new);
     }
 
+    @Transactional(readOnly = true)
+    public List<User> getNonAdminUsers() {
+        List<User> activeUsersWithIds = userRepository.findAllActivatedNonAdmins();
+        return activeUsersWithIds.stream().filter(user -> !hasAdminRole(user)).collect(Collectors.toList());
```

Comment:
Do we need to filter, even if you query for non-administrator users?

Reply:
yeah it's redundant